src/components/shared/Dropdown.css

```diff
@@ -59,12 +63,14 @@ html[dir="rtl"] .dropdown {
   mask: url(/images/prettyPrint.svg) no-repeat;
   mask-size: 100%;
   background: var(--theme-highlight-blue);
+  display: inline-block;
```

Comment:
not sure the inline-block statements have any effect, could you please check on your end?
![inline-block](https://user-images.githubusercontent.com/23530054/34588030-2d84a9b4-f1ab-11e7-8021-373f0d0ad38c.gif)